tests/utils/test_qc_metrics.py

```diff
@@ -72,13 +108,15 @@ def test_get_qc_metrics_dict(analysis_path, qc_metrics):
         "concatenated_tumor": [
             {
                 "name": "MEAN_INSERT_SIZE",
+                "norm": None,
+                "threshold": None,
                 "value": 74.182602,
-                "condition": None,
             },
             {
                 "name": "MEAN_TARGET_COVERAGE",
```

Comment:
```suggestion
                "name": "MEDIAN_TARGET_COVERAGE",
```
Analysis QC criteria are primarily based on `median_target_coverage`